endpoints/bookstore-grpc/bookstore_client.py

```diff
@@ -22,13 +22,17 @@
 import grpc
 
 
-def run(host, port, api_key, timeout):
+def run(host, port, api_key, auth_token, timeout):
     """Makes a basic ListShelves call against a gRPC Bookstore server."""
 
     channel = grpc.insecure_channel('{}:{}'.format(host, port))
 
     stub = bookstore_pb2.BookstoreStub(channel)
-    metadata = [('x-api-key', api_key)] if api_key else None
+    metadata = []
+    if api_key:
+        metadata.append(('x-api-key', api_key))
+    if auth_token:
+        metadata.append(('authorization', 'Bearer ' + auth_token))
```

Comment:
Before if there was no API key, metadata was None, now it will be an empty list. I have no idea if this matters or not,  but thought I would check